### DISCUSSION TITLE
Live diplay turning off auto_refresh doesn't work

### DISCUSSION THREAD

**Discussion by MrBeeMovie** [Textualize/rich#2096] [Q&A] [answered]
It doesn't seem like setting auto_refresh to False in a live display actually stops refreshing the display. For example:

```
    layout = Layout()

    with Live(layout, transient=True) as live:
        # turn off refresh
        live.auto_refresh = False

        layout.split_column(
            Layout(name="top", visible=True),
            Layout(name="bottom", size=1, visible=True),
        )

        prompt = Prompt("View next item in results?", choices=["y", "n"]).make_prompt(
            default="y"
        )

        layout["top"].update("Cool Entry\n" * 10000)

        layout["bottom"].update(prompt)

        # wait for input to clear display
        input("")
```

The live display will still update with the text. Am I misinterpreting the usage of auto_refresh?

Edit: 
here is an image of the output
![output](https://user-images.githubusercontent.com/36082261/159526184-fbbb1857-b809-4e37-abd5-d674ba8eb33f.PNG)

**Comment by willmcgugan** [comment, marked as answer]
Won't work with Prompt I'm afraid. Both require exclusive access to stdout.

**Reply by MrBeeMovie** [reply]
Aw man. Thank you for the prompt response!

**Reply by MrBeeMovie** [reply]
No pun intended

**Reply by MrBeeMovie** [reply]
same problem with this code:


```
  layout = Layout()

  with Live(layout, transient=True) as live:
      # turn off refresh
      live.auto_refresh = False

      layout.split_column(
          Layout(name="top", visible=True),
          Layout(name="bottom", size=1, visible=True),
      )

      layout["top"].update("Cool Entry\n" * 10000)

      layout["bottom"].update("More text.")

      while True:
          time.sleep(10)
```

![image](https://user-images.githubusercontent.com/36082261/159539190-3492a53b-399a-469a-8a3c-3b659f51cd15.png)

**Reply by willmcgugan** [reply]
Setting the auto_refresh attribute after Live has been constructed won't disable auto refresh. It should be a parameter on `Live`
